site/_docs/reference.md

```diff
@@ -2320,6 +2320,7 @@ semantics.
 | o | CHR(integer) | Returns the character having the binary equivalent to *integer* as a CHAR value
 | o | COSH(numeric)                                  | Returns the hyperbolic cosine of *numeric*
 | m o p | CONCAT(string [, string ]*)                | Concatenates two or more strings
+| m | COMPRESS(string)                               | Compresses a string and returns the result as a binary string.
 | p | CONVERT_TIMEZONE(tz1, tz2, datetime)           | Converts the timezone of *datetime* from *tz1* to *tz2*
```

Comment:
Can we specify the compress algorithm in the doc, e.g. the zlib.

Reply:
Done :)